test/rails_app/config/routes.rb

```diff
@@ -1,6 +1,92 @@
 Rails.application.routes.draw do
+  #-------------------------------------------#
+  # App Routes
+  #-------------------------------------------#
+
+
+  #-------------------------------------------#
+  # Admin
+  #-------------------------------------------#
+  # devise_for :admin_users,
+  #   path: 'admin',
+  #   controllers: {
+  #     passwords: 'admin/devise_overrides/passwords',
+  #     sessions:  'admin/devise_overrides/sessions'
+  #   }
+
   namespace :admin do
     get '/' => 'base#index'
+    get '/bootstrap.json' => 'base#bootstrap_data'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.